srcpkgs/vpsm/INSTALL.msg

```diff
@@ -0,0 +1,6 @@
+For appropriate use, export the XBPS_DISTDIR env var
+# .zshrc
+echo "export XBPS_DISTDIR=$HOME/.void-packages" >> ~/.zshrc
+
```

Comment:
Why not just modify your script to use `xdistdir` from xtools ?

Reply:
It's used. But if the `XBPS_DISTDIR` is not the `xdistdir` return a message asking to export it. That's what happened in my tests.